rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/nodesource/dataspace/DataSpaceNodeConfigurationAgent.java

```diff
@@ -304,6 +304,7 @@ public void run() {
                     FileObject[] files = rootFO.findFiles(Selectors.EXCLUDE_SELF);
                     for (FileObject file : files) {
                         if (currentTime - file.getContent().getLastModifiedTime() > invalidationPeriod) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS134)